frontend/app/src/store/balances/getters.ts

```diff
@@ -311,5 +316,17 @@ export const getters: GetterTree<BalanceState, RotkehlchenState> = {
       .map(vault => vault.debtValue)
       .reduce((sum, debt) => sum.plus(debt), Zero);
     return { totalCollateralUsd, totalDebt };
+  },
+
+  watchers: (state: BalanceState) => {
+    return state.watchers;
```

Comment:
if we dont do any transformation there is no need to add new getters we can map the state directly

Reply:
Example?

Reply:
You don't use that getter anywhere right?

Reply:
No, it will be used when we have 1 page to allow for the centralized management of all of watchers.

Reply:
1. Like I said having a getter that returns the state, doesn't make sense to me since we can map the state directly and get done with it.
2. We really should avoid introducing things that are not needed in the current PR. If we need something in a future PR we can do it in the future PR.